src/gui/optionsdlg.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>779</width>
-    <height>591</height>
+    <width>1116</width>
+    <height>838</height>
```

Comment:
This is causing the default dialog size to be large than ever, is it intentional?
Do you mind we switch back?